specification/resources/resource-manager/Microsoft.Resources/stable/2022-09-01/resources.json

```diff
@@ -4615,6 +4615,9 @@
         },
         "parameters": {
           "type": "object",
+          "additionalProperties": {
```

Comment:
Why is this additionalProperties instead of properties?  (I'm not an expert with json schema files, I assume there's probably a reason.)

Reply:
From what I learned today, `additionalProperties` is a hint what would be the values of the object/dictionary. In this case each Indvidual parameter under `parameters`. If we would use `properties` that would constraint the keys isntead.